extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -744,21 +871,36 @@ angular.module('oppia').component('svgFilenameEditor', {
       };
 
       ctrl.onSizeChange = function() {
-        var shape = ctrl.canvas.getActiveObject();
-        var size = ctrl.fabricjsOptions.size;
-        ctrl.canvas.freeDrawingBrush.width = parseInt(
-          size.substring(0, size.length - 2));
-        var strokeWidthShapes = ['rect', 'circle', 'path', 'line', 'polyline'];
-        if (shape && strokeWidthShapes.indexOf(shape.get('type')) !== -1) {
-          shape.set({
-            strokeWidth: parseInt(size.substring(0, size.length - 2))
+        if (ctrl.drawMode === DRAW_MODE_BEZIER) {
```

Comment:
Why do we need to a special case handling for Bezier? Please explain in a comment.

Reply:
Done.